ml_model/model/classification/experiments.jsonl

```diff
@@ -0,0 +1,6 @@
+{"experiment_id": "exp1-Kendals-PBU", "features": ["warnings_checker_framework", "MIDQ (min)" ,"NM (avg)","TC (avg)","Identifiers length (avg)","Line length (max)","Comments (Visual X)","Numbers (Visual X)"], "target": "PBU", "use_SMOTE" :false}
+{"experiment_id": "exp2-Kendals-ABU50", "features": ["warnings_checker_framework", "MIDQ (min)", "CICsyn (max)", "NM (avg)","TC (avg)","TC (max)","#comparisons (avg)","Identifiers length (avg)","#conditionals (avg)","#comparisons (dft)","#conditionals (dft)","Line length (dft)","Comments (Visual X)","#nested blocks (avg)","#parameters"], "target": "ABU50", "use_SMOTE" :false}
+{"experiment_id": "exp3-Kendals-BD50", "features": ["warnings_checker_framework", "TC (max)", "#conditionals (dft)" ,"Literals (Visual X)","Literals (Visual Y)","Literals (area)","Literals/identifiers (area)","Literals/keywords (area)"], "target": "BD50", "use_SMOTE" :false }
+{"experiment_id": "exp4-Kendals-PBU", "features": ["warnings_checker_framework", "MIDQ (min)" ,"NM (avg)","TC (avg)","Identifiers length (avg)","Line length (max)","Comments (Visual X)","Numbers (Visual X)"], "target": "PBU", "use_SMOTE" :true}
+{"experiment_id": "exp5-Kendals-ABU50", "features": ["warnings_checker_framework", "MIDQ (min)", "CICsyn (max)", "NM (avg)","TC (avg)","TC (max)","#comparisons (avg)","Identifiers length (avg)","#conditionals (avg)","#comparisons (dft)","#conditionals (dft)","Line length (dft)","Comments (Visual X)","#nested blocks (avg)","#parameters"], "target": "ABU50", "use_SMOTE" :true}
+{"experiment_id": "exp6-Kendals-BD50", "features": ["warnings_checker_framework", "TC (max)", "#conditionals (dft)" ,"Literals (Visual X)","Literals (Visual Y)","Literals (area)","Literals/identifiers (area)","Literals/keywords (area)"], "target": "BD50", "use_SMOTE" :true }
```

Comment:
We need to have extra experiments:
1. using all the features (code and total warnings).
2. using all the features (code and warnings separated by tool).

Reply:
Let's add more experiments by using the features selected by the Italians and warnings:
1. total
2. separated by tool